### ISSUE TITLE
Consider cutting off some workflow steps

### ISSUE THREAD

**Issue by hectorsector** [githubtraining/write-javascript-actions#12]
There are a few steps (especially near the beginning of the course) that are specifically about workflows. We can rely on other courses that introduce workflows as prereqs, and focus on writing actions here. This will reduce the number of steps and make it easier for the learner to jump in!

**Comment by mattdavis0351** [comment]
I disagree.  Since this isn't just a learning lab course, rather a piece of a learning path that is going to be a part of a more robust offering.  Have things reiterated in this course will lessen the burden on whoever is facilitating the offering onsite
